pkg/virt-controller/watch/export/export.go

```diff
@@ -507,11 +570,94 @@ func (ctrl *VMExportController) handleIsVmSnapshot(vmExport *exportv1.VirtualMac
 	return 0, ctrl.updateVMExporVMSnapshotStatus(vmExport, pvcs, pod, service, false)
 }
 
+func (ctrl *VMExportController) handleIsVM(vmExport *exportv1.VirtualMachineExport, service *corev1.Service) (time.Duration, error) {
+	var pvcs []*corev1.PersistentVolumeClaim
+
+	inUse := false
+	vmi, exists, err := ctrl.getVmi(vmExport.Namespace, vmExport.Spec.Source.Name)
```

Comment:
Not watching/triggering reconcile on VMIs.  So let's say an export is started while a VM is shutting down.  Will not be notified when the VMI is gone